lib/source.js

```
@@ -75,27 +75,26 @@ module.exports = class CovSource {
   // given a start column and end column in absolute offsets within
   // a source file (0 - EOF), returns the relative line column positions.
   offsetToOriginalRelative (sourceMap, startCol, endCol) {
-    const lines = this.lines.filter((line, i) => {
-      return startCol <= line.endCol && endCol >= line.startCol
-    })
-    if (!lines.length) return {}
+    const [startLine, endLine] = findRange(this.lines, startCol, endCol)
+    if (!startLine) {
+      return {}
+    }
 
     const start = originalPositionTryBoth(
       sourceMap,
-      lines[0].line,
-      Math.max(0, startCol - lines[0].startCol)
+      startLine.line,
+      Math.max(0, startCol - startLine.startCol)
     )
-    let end = originalEndPositionFor(
```

I moved this down so that this method can fail fast